src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -93,9 +93,9 @@ public function getLatestChannel()
     }
 
     /**
-     * @Given the store is operating on a single "France" channel
+     * @Given the store operates on a single channel in "France"
      */
-    public function thatStoreIsOperatingOnASingleFranceChannel()
+    public function thatStoreOperatesOnASingleChannelInFrance()
```

Comment:
Can we get rid of `that` in method name? It gets outdated quite often and does not have any meaning.